src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import usersFromServer from '../../api/users';
+
+interface UserInfoProps {
+  userId: number;
+}
+
+export const UserInfo = ({ userId }: UserInfoProps) => {
+  const user = usersFromServer.find((user) => user.id === userId);
+
+  if (!user) {
+    return <span className='error'>User not found</span>
```

Comment:
Consider adding a more descriptive error message or styling to the 'User not found' span to improve user experience. This is a minor issue and does not affect functionality.